doc/developer/design/20220516_transactional_consistency.md

```diff
@@ -111,9 +111,11 @@ Write read cycles (write followed by read followed by write followed by read etc
 global timestamp to increase in an unbounded fashion.
 
 Proposal: All writes across all sessions should be blocked and added to a queue. At the end of X millisecond all pending
-writes are sent in a batch to STORAGE and committed together at the same timestamp. The commits are all assigned a
-timestamp 1 larger than the current global timestamp. Once all commits have been made durable or aborted then the global
-timestamp should be increased to timestamp of the commit. No reads should be served while the writes are committing.
+writes are sent in a batch to STORAGE and committed together at the same timestamp. The commits are all assigned the
+current global write timestamp. No reads should be served while the writes are committing.
```

Comment:
Is "No reads should be served while the writes are committing" achieved by the single threaded coord? I also don't understand why that sentence needs to be here at all.

Reply:
I think so, I was trying to make the following explicit. All reads will either be at an earlier time that the writes and see none of the writes or at an equal/larger timestamp and see all of the writes. There will be no reads at an equal timestamp that see some of the writes.

Any suggestions on making this clearer?

Reply:
Honestly I'd just remove it because all of those properties are already super obvious from all the other stuff in this and other documents?

Reply:
Ok sounds good, I've removed it.